lib/commands/device/move.ts

```diff
@@ -152,7 +163,7 @@ export default class DeviceMoveCmd extends Command {
 
 		const patterns = await import('../../utils/patterns');
 		try {
-			const application = await patterns.selectApplication(
+			const applicationId = await patterns.selectApplication(
```

Comment:
This is actually the application object, not ID, so the original variable name (`application`) was actually better...

Reply:
Fixing

Reply:
Fixed.  Must have mixed it up during refactor.